rootfs/Dockerfile

```diff
@@ -14,6 +14,8 @@
 
 FROM alpine:3.3
 
+RUN apk update && apk add ca-certificates
```

Comment:
I generally prefer adding `&& rm -rf /var/cache/apk/*`, as it helps image size down.